cylc/uiserver/handlers.py

```diff
@@ -316,30 +313,34 @@ def set_default_headers(self) -> None:
         self.set_header('Server', '')
 
     def initialize(
-        self, schema=None, middleware=None, root_value=None, pretty=False,
-        batch=False, execution_context_class=None, validation_rules=None,
-        auth=None, **kwargs
+        self,
+        schema,
+        middleware=None,
+        root_value=None,
+        pretty=False,
+        batch=False,
+        execution_context_class=None,
+        validation_rules=None,
+        auth=None,
+        **kwargs,
     ):
-        super(TornadoGraphQLHandler, self).initialize()
-
-        self.schema = schema or self.schema
-        self.auth = auth
-        if middleware is not None:
-            if isinstance(middleware, MiddlewareManager):
-                self.middleware = middleware
-            else:
-                self.middleware = list(instantiate_middleware(middleware))
-        self.root_value = root_value
-        self.pretty = pretty or self.pretty
-        self.batch = batch or self.batch
-        self.execution_context_class = (
-            execution_context_class or self.execution_context_class
+        TornadoGraphQLHandler.initialize(
+            self,
+            schema,
+            middleware=middleware,
+            root_value=root_value,
+            pretty=pretty,
+            batch=batch,
+            execution_context_class=execution_context_class,
+            validation_rules=validation_rules,
         )
```

Comment:
This is already done in `TornadoGraphQLHandler.initialize`, we just need to call it with the right args.